Docs restructuring (#61)
openshift/aws-load-balancer-operator: docs/install.md

# Installation

> This documents any required information either during installation or
> post installation to ensure the operator is successfully installed.

Reviewer: ```suggestion
This documents any required information either during installation or
post installation to ensure the operator can function correctly.
```
